docs/source/a_quick_tour.mdx

```diff
@@ -178,6 +179,23 @@ A common way to overcome this issue is to fallback on single process evaluation.
 
 This solution allows 🤗 Evaluate to perform distributed predictions, which is important for evaluation speed in distributed settings. At the same time, you can also use complex non-additive metrics without wasting valuable GPU or CPU memory.
 
+## Configuration
+
+Some metrics can be configured with additional settings. E.g. `accuracy` has the extra setting `normalize` which defaults to `True`. To change it you have two options: pass it as a keyword argument with `load` or during `compute`. With `load` the setting is changed permanently for the module while passing it to `compute` only changes it for the duration of the `compute` call.
+
+```python
+
+>>> metric = evaluate.load("accuracy", normalize=False)
```

Comment:
Not sure what the convention is for the `evaluate` docs, but it's quite handy if all the code snippets "just work", so I suggest including `import evaluate`